scripts/installer/curl_install_pypi/install

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+#---------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#---------------------------------------------------------------------------------------------
+
+#
+# Bash script to install the prompt flow
+#
+INSTALL_SCRIPT_URL="https://promptflowartifact.blob.core.windows.net/linux-install-scripts/install.py"
+INSTALL_SCRIPT_SHA256=28001c408fab1fe5d265b28b7c40d5b20ab33d66d67ba92ed0eaa5104d0d7638
```

Comment:
why need to keep a sha in here. Can we keep it along with install.py?

Reply:
This is the sha of intall.py to insure the install.py not changed by others. I think we can't keep it along with install.py?

Reply:
remove sha here